plugins/com.yakindu.sct.generator.solidity/src/com/yakindu/sct/generator/solidity/CombinedState.xtend

```diff
@@ -30,7 +30,7 @@ class CombinedState implements Template {
 
 	override content(ExecutionFlow flow, GeneratorEntry entry) {
 		'''
-			pragma solidity ^0.4.18;
+			pragma solidity ^0.5.10;
```

Comment:
Don't we have a constant for the current version of the compiler, which is bound?